lib/stripe.js

```diff
@@ -21,6 +21,10 @@ Stripe.USER_AGENT = {
 
 Stripe.USER_AGENT_SERIALIZED = null;
 
+Stripe.ERROR_CLIENT_ID_NOT_SET = 'Please set stripe.clientId or pass client_id as a parameter' +
```

Comment:
This seems like a logical place to store this, since `Error.js` is specifically for API errors. Open to opinions.

Reply:
No strong feels, but it seems kind of odd to have a single error message lying here.

In other libraries, this check and error were added to be consistent with the API key check. Since (AFAICT) stripe-node doesn't have a check for the API key, maybe the right thing to do here is to remove the client_id check too and let the server return an error.

Reply:
I'm OK with leaving it to the server to return the error when the generated URL is used. I'll revert this change unless anyone has a strong feeling that we should keep it.